modules/motions/ColorDistanceLineTrace.cpp

```diff
@@ -0,0 +1,79 @@
+/**
+ * @file   ColorDistanceLineTrace.cpp
+ * @brief  色距離指定ライントレース動作
+ * @author miyahara046 nishijima515
+ */
+
+#include "ColorDistanceLineTrace.h"
+
+ColorDistanceLineTrace::ColorDistanceLineTrace(Robot& _robot, COLOR _targetColor,
+                                               double _targetDistance, double _targetSpeed,
+                                               int _targetBrightness, const PidGain& _pidGain,
+                                               bool& _isLeftEdge)
+  : LineTrace(_robot, _targetSpeed, _targetBrightness, _pidGain, _isLeftEdge),
+    targetColor(_targetColor),
+    targetDistance(_targetDistance)
+{
+}
```

Comment:
colorCountは prepare で初期化しているとはいえ、一応あった方がいいのかな？

```suggestion
ColorDistanceLineTrace::ColorDistanceLineTrace(Robot& _robot, COLOR _targetColor,
                                               double _targetDistance, double _targetSpeed,
                                               int _targetBrightness, const PidGain& _pidGain,
                                               bool& _isLeftEdge)
  : LineTrace(_robot, _targetSpeed, _targetBrightness, _pidGain, _isLeftEdge),
    targetColor(_targetColor),
    targetDistance(_targetDistance),
    colorCount(0)
{
}
```